package-lock.json

```diff
@@ -6070,7 +6070,72 @@
             "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.12.tgz",
             "integrity": "sha512-Ggd/Ktt7E7I8pxZRbGIs7vwqAPscSESMrCSkx2FtWeqmheJgCo2R74fTsZFCifr0VTPwqRpPv17+6b8Zp7th0Q==",
             "bundleDependencies": [
-                "node-pre-gyp"
+                "node-pre-gyp",
+                "abbrev",
+                "ansi-regex",
+                "aproba",
+                "are-we-there-yet",
+                "balanced-match",
+                "brace-expansion",
+                "chownr",
+                "code-point-at",
+                "concat-map",
+                "console-control-strings",
+                "core-util-is",
+                "debug",
+                "deep-extend",
+                "delegates",
+                "detect-libc",
+                "fs-minipass",
+                "fs.realpath",
+                "gauge",
+                "glob",
+                "has-unicode",
+                "iconv-lite",
+                "ignore-walk",
+                "inflight",
+                "inherits",
+                "ini",
+                "is-fullwidth-code-point",
+                "isarray",
+                "minimatch",
+                "minimist",
+                "minipass",
+                "minizlib",
+                "mkdirp",
+                "ms",
+                "needle",
+                "nopt",
+                "npm-bundled",
+                "npm-normalize-package-bin",
+                "npm-packlist",
+                "npmlog",
+                "number-is-nan",
+                "object-assign",
+                "once",
+                "os-homedir",
+                "os-tmpdir",
+                "osenv",
+                "path-is-absolute",
+                "process-nextick-args",
+                "rc",
+                "readable-stream",
+                "rimraf",
+                "safe-buffer",
+                "safer-buffer",
+                "sax",
+                "semver",
+                "set-blocking",
+                "signal-exit",
+                "string_decoder",
+                "string-width",
+                "strip-ansi",
+                "strip-json-comments",
+                "tar",
+                "util-deprecate",
+                "wide-align",
+                "wrappy",
+                "yallist"
```

Comment:
Did we expect this to grow so much? I'm not sure what bundled dependencies are tbh.

Reply:
Nah, this is purely autogenerated. I'm not sure why it changed so much either because none of the versions of this package changed... but generally I don't understand what npm is doing to the package locks and I just let it do its thing... I reran install just to be sure and npm is sure, apparently.